datafusion/core/src/lib.rs

```diff
@@ -412,7 +413,6 @@ pub const DATAFUSION_VERSION: &str = env!("CARGO_PKG_VERSION");
 extern crate core;
 extern crate sqlparser;
 
-pub mod avro_to_arrow;
```

Comment:
Technically this is a breaking API change, but I don't expect it to be a large impact as I think the use of `avro_to_arrow` is likely small. 

If it turns out others are using it more extensively, I think we should contemplate moving it to its own crate (or maybe upstream into arrow-rs)